Multisig/UI/Transactions/TransactionViewModel/TransferTransactionViewModel.swift

```diff
@@ -104,6 +103,13 @@ class TransferTransactionViewModel: TransactionViewModel {
             } else if let address = transfer.tokenAddress?.address {
                 if let known = App.shared.tokenRegistry.token(address: address) {
                     token = known
+                    // Token type may be invalid because TokenRegistry
+                    // assumes "erc20" when it doesn't know the actual
+                    // type.
+                    // Now, with the information from the transfer, we
+                    // can enhance the cached token
+                    token.type = transfer.type == .erc20 ? .erc20 : .erc721
```

Comment:
Not sure if it's valid idea but can we define same enum for transfer type& token type?

Reply:
they have different values, we could, but in a different task

Reply:
#415